schema/codelists/awardStatus.csv

```diff
@@ -1,5 +1,6 @@
 Code,Title,Description
-pending,Pending,"This award has been proposed, but is not yet in force. This can be due to a cooling off period, or some other process."
```

Comment:
Omg, sorry. Totally forgot to deprecate instead of remove. Deprecation notes added in https://github.com/open-contracting/standard/pull/1509/commits/f5aa66f42d8c05bc4935a78be96f0850b7d1f7c4.